ecosystem/indexer-grpc/indexer-grpc-file-store-backfiller/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod metrics;
+pub mod processor;
+
+use anyhow::Result;
+use aptos_indexer_grpc_server_framework::RunnableConfig;
+use aptos_indexer_grpc_utils::config::IndexerGrpcFileStoreConfig;
+use processor::Processor;
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct IndexerGrpcFileStoreBackfillerConfig {
+    pub file_store_config: IndexerGrpcFileStoreConfig,
+    pub fullnode_grpc_address: Url,
+    pub progress_file_path: String,
+    pub enable_expensive_logging: Option<bool>,
+    pub chain_id: u64,
+    #[serde(default = "default_enable_cache_compression")]
+    pub enable_cache_compression: bool,
+    pub starting_version: Option<u64>,
+    pub transactions_count: Option<u64>,
+}
+
+const fn default_enable_cache_compression() -> bool {
+    false
```

Comment:
Why is this false? I thought the default is true

Reply:
good question: for new components we should default this to true; updated.

Reply:
is this still valid?

Reply:
sort of; there is no survey yet about if anyone still runs under uncompressed format.